src/Microsoft.ML.ImageAnalytics/ImageLoaderTransform.cs

```diff
@@ -239,5 +240,61 @@ public override SchemaShape GetOutputSchema(SchemaShape inputSchema)
 
             return new SchemaShape(result.Values);
         }
+
+        internal sealed class OutPipelineColumn : Scalar<UnknownSizeBitmap>
+        {
+            private readonly Scalar<string> _input;
+
+            public OutPipelineColumn(Scalar<string> path, string relativeTo)
+                : base(new Reconciler(relativeTo), path)
+            {
+                Contracts.AssertValue(path);
+                _input = path;
+            }
+
+            /// <summary>
+            /// Reconciler to an <see cref="ImageLoaderEstimator"/> for the <see cref="PipelineColumn"/>.
+            /// </summary>
+            /// <remarks>
+            /// We must create a new reconciler per call, because the relative path of <see cref="ImageLoaderTransform.Arguments.ImageFolder"/>
+            /// is considered a transform-wide option, as it is not specified in <see cref="ImageLoaderTransform.Column"/>. However, we still
```

Comment:
>as it is not specified in <s [](start = 55, length = 28)

should we fix the comments in ImageLoaderTransform.Column and remove the 'not' from here?

Reply:
It's not a comment problem, is it? The setting simply isn't there. Frankly I don't really consider the issue, if we agree it is an issue which I'm not sure I do, to be sufficiently interesting as to be worth solving. I mean, seriously, it only affects situations where you're loading multiple images per column, so I have a hard time getting excited about it.

---
In reply to: [214722459](https://github.com/dotnet/machinelearning/pull/778#discussion_r214722459) [](ancestors = 214722459)